diff --git a/node_modules/@sentry/node/cjs/handlers.js b/node_modules/@sentry/node/cjs/handlers.js
index 2682a5e..edc3a1f 100644
--- a/node_modules/@sentry/node/cjs/handlers.js
+++ b/node_modules/@sentry/node/cjs/handlers.js
@@ -364,7 +364,8 @@ function requestHandler(
     var local = domain.create();
     local.add(req);
     local.add(res);
-    local.on('error', next);
+    // Kirill: skipping this on-error call to avoid breaking user-requests with `void` functions
+    // local.on('error', next);
 
     local.run(() => {
       var currentHub = core.getCurrentHub();
